functions.php

```diff
@@ -19,6 +19,20 @@ function getAllStudents($db, $active, $studentID)
 	}
 	return $myOutput;
 }
+
+//get student ID of user
+function getStudentID($db, $userID)
+{
+	$query = "SELECT * from `student`";
+	$result = $db->query($query) or error_log("\n<br />Warning: query failed:$query. " . $db->error. ". At file:". __FILE__ ." by " . $_SERVER['REMOTE_ADDR'] .".");
+	while ($row = $result->fetch_assoc()):
```

Comment:
Instead of searching through the entire table with php you should change your query:
"SELECT * from `student` WHERE  `userID` = '$userID'"

Reply:
Done.